java/integTest/src/test/java/glide/PubSubTests.java

```diff
@@ -130,27 +136,54 @@ public void cleanup() {
         pubsubMessageQueue.clear();
     }
 
+    private enum MessageReadMethod {
```

Comment:
There was a TODO to about tests relating to async right?

Reply:
Yes, it was. Implemented somewhere there.